### PR TITLE
Add a new option that removes "density| word from Planet Wizard

### PR DESCRIPTION
**Pull request by pjanik**
https://www.pivotaltracker.com/story/show/162387016

New option + text changes.
Also, I've updated test libs, added Enzyme lib, configured it and added the first unit test of a component.

### CODE REVIEW

File: js/components/sortable-densities.js [hunk 1]
```diff
@@ -77,11 +78,17 @@ export default class SortableDensities extends Component {
...
+          { config.densityWordInPlanetWizard ? 'LOW' : 'ABOVE' }
           <SortableList plateInfos={this.plateInfos} onSortEnd={this.onSortEnd} useDragHandle={false} />
-          HIGH
+          { config.densityWordInPlanetWizard ? 'HIGH' : 'BELOW' }
+        </div>
+        <div className='helper-text'>
+          {
+            config.densityWordInPlanetWizard
+              ? 'Click and drag to reorder the plate density'
+              : 'Click and drag to reorder the plate that will go below the other plate'
```
Comment (ekosmin) [inline js/components/sortable-densities.js]: Is this the desired wording? I'd think something like "Click and drag to order the plates from top to bottom" is more clear, especially when there are more than 2 plates

File: test/components/sortable-densities.test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+const store = {
+  model: {
+    plates: []
+  }
+}
+
+describe('SortableDensities component', () => {
+  beforeEach(() => {
+    // Restore config as it can be modified by tests.
+    Object.assign(config, originalConfig)
```
Comment (ekosmin) [inline test/components/sortable-densities.test.js]: To fully restore the config, including new keys, would you want to do something like `Object.assign({}, originalConfig)` instead?

Comment (pjanik) [reply test/components/sortable-densities.test.js]: Good point, thanks!

File: js/components/sortable-densities.js (hunk 1 above)
Comment (pjanik) [reply js/components/sortable-densities.js]: I've used wording provided by Amy. I'll double check with her.

Comment (pjanik) [reply js/components/sortable-densities.js]: Amy agreed that this prev version was confusing. She provided another variant that I used in the last commit.

File: test/components/sortable-densities.test.js (hunk 2 above)
Comment (pjanik) [reply test/components/sortable-densities.test.js]: That wouldn't work actually, as it's not enough to overwrite local `config` reference. I've added another approach to delete new keys. If we use this pattern more, I guess we should move it to some helper method.

**Pull request merged**